docker/config/slick.sh-dist

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+#
+# $Id$
+#
+# Bootstrap script for a Tecken local development environment. Refer to
+# documentation of the local dev environment and some of these commands at
+# https://tecken.readthedocs.io/
+
+set -ex
+
+# Change these to whatever you need. These are fake credentials to be used in a
+# local development environment--not a server environment.
+#
+# Fill these in
+FAKEUSERNAME=""
+FAKEPASSWORD=""
+FAKEEMAIL=""
+
+# 32-character hex token
+FAKETOKEN="c7c1f8cab79545b6a06bc4122f0eb3cb"
+
+# Build Tecken
+make build
+
+# Start services
+docker compose up -d --remove-orphans db redis-cache statsd localstack oidcprovider fakesentry
```

Comment:
I have this line in my `slick.sh` script because I've had a lot of problems with race conditions and other things, so having this line makes it more likely that all the backing services have fully started. We could add `waitfor` lines to `make setup` so it starts services and then waits to make sure they're started up. That would probably alleviate some of the problems I had, but it's probably easier to just do this.

Plus it has the `--remove-orphans` flag which removes orphaned services.

Reply:
We can take it out and do any other improvements in follow-up PRs.

Reply:
In that case, I think the right solution is to bake this into the Docker Compose config, using health checks and dependencies to make sure the services are ready before a web container is started. Maybe at some point in the future, doesn't seem urgent.